### DISCUSSION TITLE
Feedback for special symbols in `usage` messages

### DISCUSSION THREAD

**Discussion by kenkangxgwe** [kenkangxgwe/lsp-wl#71] [Ideas]
This issue keeps track of the symbols that are not shown correctly in the symbol usage messages (such as hover and completion). 

Some symbols are defined in the Private Use Area of Mathematica fonts, thus we are looking for Unicode symbols to replace them.

For example:
`\[LeftAssociation]` -> `<|`
`\[TwoWayRule]` -> `<->`

TODO:
- [x] ~`ReplacePart::usage`~: `"\[Null]" -> ""`, fixed in e778be7
- [x] ~`BeginPackage`~: ``"`" -> "\`"``, fixed in 5bb09a4
- [x] ~`Root`~: `"\[Equal]" -> "=="`, fixed in e778be7

**Comment by minkir014** [comment]
If you have finished the missions. So close this issue.

**Comment by kenkangxgwe** [comment]
Hi @minkir014 , this is a long-term issue that will keep adding new TODOs whenever we find new bugs in the symbol definition. : ). Feel free to add one if you encounter.
